Dockerfile

```diff
@@ -8,14 +8,15 @@ CMD ["/app/entrypoint.sh", "start"]
 
 # Git commit SHAs for the build artifacts we want to grab.
 # From https://github.com/phacility/phabricator
-# Promote 2019 Week 8
-ENV PHABRICATOR_GIT_SHA 5d88eef26b72623be528f6e448b5615a7cb166c2
+# Promote 2019 Week 12
+# Plus two extra commits
+ENV PHABRICATOR_GIT_SHA e51381e2d6fa215d8be94708322316090ed741d2
 # From https://github.com/phacility/arcanist
 # Promote 2019 Week 8
 ENV ARCANIST_GIT_SHA b4a302683b1aefbbb2ab9d1aaaf418b551b84b28
 # From https://github.com/phacility/libphutil
```

Comment:
It would be good to update this to https://github.com/phacility/libphutil/tree/stable